middleware.go

```diff
@@ -207,7 +218,9 @@ func (m retryMiddleware) RoundTrip(req *http.Request) (*http.Response, error) {
 		Attempts: m.policy.Attempts,
 		Delay:    m.policy.Delay,
 		BackoffFunc: func(delay time.Duration, attempts int) time.Duration {
-			return m.defaultBackoff(res, delay)
+			var duration time.Duration
+			duration, backOffErr = m.defaultBackoff(res, delay)
```

Comment:
where is backOffErr evaluated? or logged?

Reply:
https://github.com/juju/http/pull/6/files#diff-02f50901140d6057da6310a106670552aa766a093efbc2200fb34c099b762131R200-R202